pennylane_lightning/lightning_qubit.py

```diff
@@ -17,8 +17,15 @@
 """
 from typing import List
 from warnings import warn
-from os import getenv
 from itertools import islice
+import platform
+from os import getenv, environ
+
+if platform.system() == "Linux":
+    # Kokkos ask to set OMP_PROC_BIND=spread and OMP_PLACES=threads
+    # for general best performance with OpenMP 4.0 or later.
+    environ["OMP_PROC_BIND"] = "spread"
+    environ["OMP_PLACES"] = "threads"
```

Comment:
I do not think we want to set these variables. Those are for the user to decide.

Reply:
OK. I will remove it. Without it, the user will keep getting a Kokkos warning saying that these variables must be set for better performance.

Reply:
That is fine. I think it is bad practice of Kokkos to expect this, so we should assume it will go away at some point. Putting the responsibility on the user for this is a better design.

If it becomes problematic later, we can check for existence of such variables and only set if they do not exist, but for now I think keeping this off is fine.

Reply:
We can set it back later if we find it a good idea.

Reply:
> 

Sure! Good idea!